appsec/src/extension/tags.c

```diff
@@ -40,6 +42,7 @@
 #define DD_TAG_HTTP_RH_CONTENT_LANGUAGE "http.response.headers.content-language"
 #define DD_TAG_HTTP_CLIENT_IP "http.client_ip"
 #define DD_TAG_USER_ID "usr.id"
+#define DD_TAG_SERVER_REQUEST_PATH_PARAMS "server.request.path_params"
```

Comment:
I'm not entirely sure why this is here, seems to be duplicated.

Reply:
Deleted